sites/public/src/components/listing/ListingViewSeeds.tsx

```diff
@@ -0,0 +1,752 @@
+  const googleMapsHref =
+    "https://www.google.com/maps/place/" + oneLineAddress(listing.listingsBuildingAddress)
+  const applicationsClosed = dayjs() > dayjs(listing.applicationDueDate)
+  const amiValues = getAmiValues(listing)
+  const redirectIfSignedOut = () =>
+    process.env.showMandatedAccounts && initialStateLoaded && !profile
+  const onlineApplicationUrl = redirectIfSignedOut()
+    ? `/sign-in?redirectUrl=/applications/start/choose-language&listingId=${listing.id}`
+    : getOnlineApplicationURL(listing.applicationMethods, listing.id, props.preview)
+  const disableApplyButton = !props.preview && listing.status !== ListingsStatusEnum.active
+  const eligibilitySections = getEligibilitySections(listing)
+  const paperApplications = getPaperApplications(listing.applicationMethods)
+  const paperApplicationURL: string = watch(
+    "paperApplicationLanguage",
+    paperApplications?.length ? paperApplications[0].fileURL : undefined
+  )
+  const hasPaperApplication =
+    !!getMethod(listing.applicationMethods, ApplicationMethodsTypeEnum.FileDownload) &&
+    paperApplications.length > 0
+
+  const applicationPickUpAddress = getAddress(
+    listing.applicationPickUpAddressType,
+    "pickUp",
+    listing
+  )
+  const applicationMailingAddress = getAddress(
+    listing.applicationMailingAddressType,
+    "mailIn",
+    listing
+  )
+  const applicationDropOffAddress = getAddress(
+    listing.applicationDropOffAddressType,
+    "dropOff",
+    listing
+  )
+  const postmarkString = getPostmarkString(
+    listing.applicationDueDate
+      ? getDateString(listing.applicationDueDate, `MMM DD, YYYY [${t("t.at")}] hh:mm A`)
+      : null,
+    listing.postmarkedApplicationsReceivedByDate
+      ? getDateString(
+          listing.postmarkedApplicationsReceivedByDate,
+          `MMM DD, YYYY [${t("t.at")}] hh:mm A`
+        )
+      : null,
+    listing.developer
+  )
+  const openHouseEvents = listing.listingEvents
+    ?.filter((event) => event.type === ListingEventsTypeEnum.openHouse)
+    .map((event) => getEvent(event))
+  const publicLotteryEvent = listing.listingEvents?.find(
+    (event) => event.type === ListingEventsTypeEnum.publicLottery
+  )
+  const lotteryResultsEvent = listing.listingEvents?.find(
+    (event) => event.type === ListingEventsTypeEnum.lotteryResults
+  )
+  const lotteryRanNoResultsPosted =
+    dayjs(publicLotteryEvent?.startTime) < dayjs() && !lotteryResultsEvent
+  const listingTags = getListingTags(listing)
+  const features = getFeatures(listing, props?.jurisdiction)
+  const lotteryResultsPdfUrl = pdfUrlFromListingEvents(
+    [lotteryResultsEvent],
+    ListingEventsTypeEnum.lotteryResults,
+    process.env.cloudinaryCloudName
+  )
+  const enableUtilitiesIncluded = props.jurisdiction.featureFlags?.some(
+    (flag) => flag.name === "enableUtilitiesIncluded" && flag.active
+  )
+  const utilitiesIncluded = getUtilitiesIncluded(listing)
+
+  // Right bar sections ----------
+  const DueDate = (
```

Comment:
suggestion: if we exported these "section" functions we could more easily test the page by writing individual tests for each section and then not have to render the whole page to verify all of the content is there. We would still want tests for the page level, but those tests could just verify that each section appears

Reply:
Totally hear that! I think I hesitated at first because I had to pass so much data around when I separated the sections out. I'll take a pass at separating them into separate files / components.

Reply:
We can also do that as a follow-up when we get to the tests for this page

Reply:
I separated out the sections into components - at first some of the componentization bordered on feeling like overkill but it made it wayyyyyy easier to test, andto read that main file!